Header header
can_wrapper/Stepper[6] axes
can_wrapper/Stepper gripper